packages/fuels/test/fixtures/project/.fuels/chainConfig.json

```diff
@@ -131,8 +131,20 @@
           "xori": 2,
           "alocDependentCost": {
             "LightOperation": {
-              "base": "2",
-              "unitsPerGas": "214"
+              "base": 2,
+              "unitsPerGas": 214
+            }
+          },
```

Comment:
Seems like the `alocDependentCost` is defined as a `HeavyOperation` on the Testnet now, and using different values. Should we match the values for the other gas costs as well?